meson.build

```diff
@@ -646,6 +646,10 @@ if get_option('build_backends')
     has_backends = true
     add_project_arguments('-fobjc-arc', language : 'objc')
     add_project_arguments('-fobjc-arc', language : 'objcpp')
+
+    # Add new Acccelerate headers in MacOs >= 13.3. This may fail in older versions.
```

Comment:
Why would defining a few symbols fail? Are there bad versions in earlier MacOS versions?

Reply:
We now get a new warning on circleci:
> ld: warning: object file (/tmp/lto.o) was built for newer macOS version (12.6) than being linked (12.0)

Is there an equivalent linker flag that needs setting or is it a build image issue?